site/src/css/components/_Carousel.scss

```diff
@@ -0,0 +1,158 @@
+/* Carousel CSS */
+
+@import './modules/CarouselSlide';
+
+.carousel-container {
+    width: 100%;
+    height: 550px;
+    display: flex;
+    flex-direction: column;
+    margin: 2em 0;
+
+    @include mobile-view {
+        height: 300px;
+        margin: 0;
+    }
+    @include ultra-slim-mobile-view {
+        height: 240px;
+    }
+
+    .decoration {
+        height: 50px;
+        width: 100%;
+        display: flex;
+
+        @include mobile-view {
+            display: none;
+        }
+    }
+    
+    .top {
+        margin-bottom: -1px;
+
+        svg {
+            transform: scaleY(-1);
+            width: 200px;
+            margin-left: 70%;
+        }
+    }
+    
+    .bottom {
+        margin-top: -1px;
+
+        svg {
+            width: 200px;
+            margin-right: 70%;
+            margin-left: auto;
+        }
+    }
+
+    .content {
+        width: 100%;
+        height: 450px;
+        background-color: var(--carousel-color);
+        overflow: hidden;
+        display: flex;
+        flex-direction: column;
+        justify-content: space-evenly;
+
+        @include mobile-view {
+            height: 100%;
+        }
+
+        .carousel-header {
+            width: auto;
+            height: 50px;
+            color: var(--ifm-font-color-base);
+            font-family: 'Overpass Bold';
+            font-size: 32px;
+            display: flex;
+            justify-content: center;
+            align-items: center;
+
+            @include mobile-view {
+                height: 30px;
+                font-size: 24px;
+            }
+            @include slim-mobile-view {
+                height: 22px;
+                font-size: 20px;
+            }
+            @include ultra-slim-mobile-view {
+                height: 20px;
+                font-size: 18px;
+            }
```

Comment:
should height be a `%` and font be `em` 

Im not sure what this controls

Reply:
Responsiveness is a lot easier if either % or `em` or `rem` is used throughout.